src/test/java/htsjdk/tribble/readers/AsciiLineReaderTest.java

```diff
@@ -49,4 +41,20 @@ public void testReadLines() throws Exception {
         assertEquals(expectedNumber, actualLines);
 
     }
+
+    @Test public void voidTestLineEndingLength() throws Exception {
+        final String input = "Hello\nThis\rIs A Silly Test\r\nSo There";
+        final InputStream is = new ByteArrayInputStream(input.getBytes());
```

Comment:
I'm not asking to modify the valid line-endings...I just want the tests to be explicit about it.

So the test will show an empty line and the correct size of line ending.